features/in_onderzoek.feature

```diff
@@ -12,109 +12,235 @@ Functionaliteit: in onderzoek
-      | 5         | 555550002                   | Karel     | M                           | 20091102              |                          |                                 |                                |                               |
+    Scenario: vragen om naam bij hele categorie persoon in onderzoek
+      Gegeven een persoon heeft de volgende 'persoon' gegevens
+      | burgerservicenummer (01.20) | voornamen (02.10) | geboortedatum (03.10) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) | 
+      | 000000097                   | Arnitta           | 19231213              | 010000                          | 20120920                       |
+      En de persoon heeft een ouder '1' met de volgende gegevens
+      | voornamen (02.10) | geslachtsaanduiding (04.10) |
+      | Noa               | V                           |
+      En de persoon heeft een ouder '2' met de volgende gegevens
+      | voornamen (02.10) | geslachtsaanduiding (04.10) |
+      | Jan               | M                           |
+      En de persoon heeft een 'partner' met de volgende gegevens
+      | burgerservicenummer (01.20) | voornamen (02.10 | geslachtsaanduiding (04.10) | datum aangaan (06.10) |
+      | 000000103                   | Karel            | M                           | 20091102              |
       Als personen wordt gezocht met de volgende parameters
       | naam                | waarde                          |
       | type                | RaadpleegMetBurgerservicenummer |
-      | burgerservicenummer | 555550001                       |
-      | fields              | ouders,partners,<fields>        |
-      Dan heeft de persoon de volgende 'persoonInOnderzoek' gegevens
-      | naam                          | waarde   |
-      | aanduidingGegevensInOnderzoek | <waarde> |
-      | datumIngangOnderzoek          | 20120920 |
-      En heeft de persoon GEEN 'naam.inOnderzoek' gegevens
-      En heeft de persoon GEEN 'geboorte.inOnderzoek' gegevens
-      En heeft de persoon GEEN 'verblijfplaats' gegevens
-      En heeft de ouder met ouderAanduiding '1' GEEN 'inOnderzoek' gegevens
-      En heeft de ouder met ouderAanduiding '2' GEEN 'inOnderzoek' gegevens
-      En heeft de partner met burgerservicenummer '555550002' GEEN 'inOnderzoek' gegevens
+      | burgerservicenummer | 000000097                       |
+      | fields              | ouders,partners,naam            |
+      Dan heeft de response een persoon met de volgende 'naam' gegevens
+      | naam                      | waarde  |
+      | voornamen                 | Arnitta |
+  	  | voorletters               | A.      |
+      | volledigeNaam             | Arnitta |
+      | inOnderzoek.voornamen     | true    |
+      | inOnderzoek.voorletters   | true    |
+      | inOnderzoek.volledigeNaam | true    |
```

Comment:
dit is niet compleet. bijv. inOnderzoek.voorvoegsel en inOnderzoek.datumIngangOnderzoek

Reply:
Ben ik daarmee dan compleet of moeten er nog veel meer properties in onderzoek worden gezet? De gehele categorie is immers in onderzoek gezet. Dus moeten ook geslacht, leeftijd en ook datum, land en plaats binnen geboorte en overlijden in onderzoek worden gezet?.

Reply:
Gezien je opmerking bij regel 137 is dit niet het geval. Daaruit trek ik de conclusie dat alleen van velden die gevraagd worden indien van toepassing de in onderzoek meegegeven moeten worden.

Reply:
Is aangepast